tests/e2e/etcd_process.go

```diff
@@ -153,3 +160,10 @@ func (ep *etcdServerProcess) waitReady() error {
 }
 
 func (ep *etcdServerProcess) Config() *etcdServerProcessConfig { return ep.cfg }
+
+func (ep *etcdServerProcess) Logs() logsExpect {
+	if ep.proc == nil {
+		panic("please grap logs before process is stopped")
```

Comment:
Nit: Is this meant to say `grab`?

Reply:
aha, typo is inherited from https://github.com/etcd-io/etcd/blob/fd0882b67e6f5238cbee9a75688537864c0deb5c/tests/framework/e2e/etcd_process.go#L261.

I assume it meant grab and fixed.

Reply:
Pls feel free to raise a PR to fix the typo in main branch :)